### ISSUE TITLE
Can't reproduce the results with sample run provided for CIFAR-10

### ISSUE THREAD

**Issue by feat7** [hongyi-zhang/mixup#7]
I ran the command as provided in readme:

```bash
python easy_mixup.py --sess my_session_1 --seed 11111
```

Since, I am using pytorch >= 1.0
I had to fix the loss calculation:

`train_loss += loss.data[0]` to `train_loss += loss.data.item()`
and `test_loss += loss.data[0]` to `test_loss += loss.data.item()`

The maximum accuracy I got was 93.90% which translates to an error of 6.1

What am I doing wrong here?
